CMakeLists.txt

```diff
@@ -606,15 +606,21 @@ if (BUILD_TESTING)
             set(FUZZ_TIMEOUT_SEC 60)
         endif()
 
-        if(NOT DEFINED CORPUS_UPLOAD_LOC)
+        if(DEFINED ENV{CORPUS_UPLOAD_LOC})
+            set(CORPUS_UPLOAD_LOC $ENV{CORPUS_UPLOAD_LOC})
+        else()
             set(CORPUS_UPLOAD_LOC "none")
         endif()
 
-        if(NOT DEFINED ARTIFACT_UPLOAD_LOC)
+        if(DEFINED ENV{ARTIFACT_UPLOAD_LOC})
+            set(ARTIFACT_UPLOAD_LOC $ENV{ARTIFACT_UPLOAD_LOC})
```

Comment:
`CORPUS_UPLOAD_LOC` and `ARTIFACT_UPDLOAD_LOC` are set as environment vars in the codebuild job?

Reply:
Yes, they are defined on CodeBuild as env vars because we don't want to expose these to public.